### PR TITLE
Druid Issue #1914: Added an injection test for ZkCoordinator.

### PR DESCRIPTION
**Pull request by liuqiyun**
I added a test for testing the Inject annotation on ZkCoordinator. In this test, the instance of ZkCoordinator is generated by Guice Injector. And this test could pass permanently.

### CODE REVIEW

**Comment by fjy** [comment]
:+1

File: server/src/test/java/io/druid/server/coordination/ZkCoordinatorTest.java [hunk 1]
```diff
@@ -283,4 +291,84 @@ private void checkCache(List<DataSegment> segments) throws IOException
       Assert.assertEquals(segments.get(i), segment);
     }
   }
+  
+  @Test
+  public void testInjector() throws Exception
+  {
+    Injector injector = Guice.createInjector(new Module() {
```
Comment (drcrallen) [inline server/src/test/java/io/druid/server/coordination/ZkCoordinatorTest.java]: Usually injectors are created in a method similar to https://github.com/druid-io/druid/blob/master/server/src/test/java/io/druid/client/cache/MemcachedCacheTest.java#L144  

where 
```binder.bindConstant().annotatedWith(Names.named("serviceName")).to("druid/test/memcached");
binder.bindConstant().annotatedWith(Names.named("servicePort")).to(0);```
is used inside of a call to `Initialization.makeInjectorWithModules( GuiceInjectors.makeStartupInjector(), ImmutableList.of(`

Is that method able to be used here?

**Comment by gianm** [comment]
Thanks @liuqiyun! This generally looks good to me.

Could you please:

- apply the Druid style jar (intellij_formatting.jar in the repo root)
- fill out the CLA at http://druid.io/community/cla.html, unless your company has already filled out a corporate one (in which case please tell us which company that is)

**Comment by liuqiyun** [comment]
@gianm As your comments:
- I applied the Druid style jar (intellij_formatting.jar) in my IDE and regenerated a commit with the expected format. And then, I pushed the new commits to my regarding branch 1914-v2
- Filled out the CLA

Thank you!

**Comment by nishantmonu51** [comment]
+1, I am in after squashing commits.

**Comment by gianm** [comment]
@liuqiyun you can squash commits using the method on https://github.com/druid-io/druid/blob/master/CONTRIBUTING.md

**Comment by liuqiyun** [comment]
@gianm thanks for reminder :-) just squashed commits into one and please let me know if need any further actions

**Comment by fjy** [comment]
@liuqiyun do you mind signing the CLA at druid.io/community/cla.html?

I can merge things after

**Comment by liuqiyun** [comment]
@fjy just sent my CLA in pdf format to druid-admin@metamarkets.com :)

**Pull request merged**